spec/requests/users_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+    let(:valid_attributes) {
+        {email: 'john@example.com', password: 'test_password', password_confirmation: 'test_password'} 
+    }
+
+    let(:invalid_attributes) {
+        {email: 'john@example.com', password: 'test_password', password_confirmation: 'unmatching_test_password'} 
+    }
+
+    describe "GET /new" do
```

Comment:
This would be `GET /users/new` if we are testing with `get new_user_url`, but the github comment above states that the more user-friendly way to create an account is through the `GET /sign_up` URL. We aren't testing that route though.

I would expect `describe "GET /sign_up" do`. Then we would need to use the `get sign_up_url` to make the request.